src/App.css

```diff
@@ -1,106 +1,203 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;700&family=Roboto:wght@900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,700;1,100;1,200;1,300&display=swap');
+
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
   list-style: none;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+a {
+  font-family: Roboto Slab, sans-serif;
+  color: #4386bf;
 }
 
 body {
-  background-color: #f1f1f1;
+  background-color: #fafafa;
+  font-family: 'Roboto', sans-serif;
 }
 
-.newbook {
+nav > button {
+  margin-left: auto;
+}
+
+form {
   display: flex;
-  margin: auto;
-  flex-direction: column;
-  justify-content: center;
   align-items: center;
   gap: 1em;
 }
 
+.newbook {
+  border-top: solid 1px #e8e8e8;
+  margin-left: 1%;
+  padding-bottom: 14px;
+  width: 100%;
+  margin-top: 3%;
+}
+
+.newbook > h3 {
+  font-size: 1.25rem;
+  font-family: 'Montserrat', sans-serif;
+  height: 1rem;
+  width: 50%;
+  margin: 1.813rem 32.75rem 1.188rem 0;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
 .newbook button {
-  margin-top: 10px;
   font-size: 12px;
-  background-color: rgb(25, 173, 192);
+  background-color: rgb(2, 144, 255);
   color: white;
   border: none;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  width: 18%;
+  height: 2.713rem;
+  font-family: Roboto Slab, sans-serif;
+  font-weight: bold;
+  letter-spacing: 0.1em;
+}
+
+input {
+  width: 30%;
+  height: 2.513rem;
   padding: 5px;
-  border-radius: 5px;
+  font-size: 1rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.15px;
+  font-family: Montserrat, sans-serif;
+  border: 1px solid #e8e8e8;
+  outline: none;
+}
+
+input::placeholder {
+  color: #d4d4d4;
+}
+
```

Comment:
| @Katended Design | Zeplin's Design |
|-|-|
![image](https://github.com/Katended/book-store/assets/93039359/e271135d-eb20-47ae-abfa-dfe703fbb504)
![image](https://github.com/Katended/book-store/assets/93039359/7c3f6068-66cf-4b15-94f2-35ed89fb8cb2)

- [ ] Kudos on implementing the form request changes but I guess while doing that you have made some mistakes 😨 It would be nice if you make your form expand as you can see in the given design to make your project follow the requirements 👍